module/userProjects/helper.js

```diff
@@ -1293,6 +1349,26 @@ module.exports = class UserProjectsHelper {
 
 							projectCreation.data['solutionExternalId'] =
 								projectCreation.data.solutionInformation.externalId
+
+							// If req.body contains acl add it, else add defaultACL
+							if (bodyData.hasOwnProperty('acl')) {
```

Comment:
@Prajwal17Tunerlabs why we have to add acl while sync the project if it is not in the req?

Reply:
im adding default acl if acl is not present in the req.